app/src/androidTest/java/co/nimblehq/compose/crypto/ui/screen/detail/DetailScreenUiTest.kt

```diff
@@ -0,0 +1,146 @@
+package co.nimblehq.compose.crypto.ui.screen.detail
```

Comment:
@kaungkhantsoe Would you mind create a package for `home` and move `HomeScreenUITest` inside there 🙇‍♀️